sources/europe/Tours-Orthophoto-2013.json

```diff
@@ -0,0 +1,22 @@
+{
+    "url": "http://wms.openstreetmap.fr/tms/1.0.0/tours_2013/{zoom}/{x}/{y}", 
+    "attribution": {
+        "url": "http://wiki.openstreetmap.org/wiki/Tours/Orthophoto", 
+        "text": "Orthophoto Tour(s)plus 2013", 
+        "required": true
+    }, 
+    "type": "tms",
+    "name": "Tours - Orthophotos 2013",
+    "start_date": "2008",
+    "end_date": "2013",
+    "country_code": "FR",
+    "extent": {
+        "max_zoom": 22,
```

Comment:
the zoom 22 is on the tms server, you can have a look at http://wms.openstreetmap.fr/tms/1.0.0/tours_2013/. I did not compute tiles, I don't know more. Sorry